Mathlib/Data/Finset/Pointwise.lean

```diff
@@ -2208,8 +2241,57 @@ protected theorem neg_smul [DecidableEq α] : -s • t = -(s • t) := by
 
 end Ring
 
+section BigOps
+section CommMonoid
+variable [CommMonoid α] {ι : Type*} [DecidableEq ι]
+
+@[to_additive (attr := simp)] lemma prod_inv_index [InvolutiveInv ι] (s : Finset ι) (f : ι → α) :
```

Comment:
Did you verify with `to_additive?` that it doesn't change `s⁻¹` to `-s` here?

Reply:
It does, which is why I have extra lemmas below.